src/createReducer.js

```diff
@@ -361,6 +361,11 @@ function createReducer<M, L>(structure: Structure<M, L>) {
         result = setIn(result, 'values', values)
         result = setIn(result, 'initial', values)
       }
+      var submitFailed = getIn(state, 'submitFailed');
```

Comment:
If the user is resetting changes, why would they want failures to be propagated? What's the use case for this?